ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -26,6 +26,7 @@ StackView {
     // General properties:
     property int viewWidth: 560 // by design
     property string previousPageName: depth > 1 ? qsTr("Back") : ""
+    required property string communityId
```

Comment:
This will break the respective storybook page (which you hadn't adjusted here in this PR). Yes, we will have a CI check for this soon

Reply:
Fixed.